src/translations/en.json

```diff
@@ -2048,11 +2048,29 @@
                   "conversation_engine": "Conversation agent",
                   "conversation_language": "[%key:ui::panel::config::voice_assistants::assistants::pipeline::detail::form::language%]",
                   "language": "Language",
-                  "stt_engine": "Speech to text",
+                  "stt_engine": "Speech-to-text",
                   "stt_language": "[%key:ui::panel::config::voice_assistants::assistants::pipeline::detail::form::language%]",
-                  "tts_engine": "Text to speech",
+                  "tts_engine": "Text-to-speech",
                   "tts_language": "[%key:ui::panel::config::voice_assistants::assistants::pipeline::detail::form::language%]",
                   "tts_voice": "Voice"
+                },
+                "steps": {
+                  "config": {
+                    "title": "Configuration",
+                    "description": "Main configuration of your assistant."
+                  },
+                  "conversation": {
+                    "title": "Conversation agent",
+                    "description": "The conversation agent is the brains of your voice assistant and will process the incoming commands."
+                  },
+                  "stt": {
+                    "title": "Speech-to-text",
+                    "description": "When you are using the pipeline as a voice assistant, the speech-to-text engine turns your voice command into text."
```

Comment:
Do we refer to this as an assistant or as a pipeline? If just assistant, maybe this: "When you are controlling your assistant with voice..."

Reply:
👍  We don't refer to pipeline in the UI.